src/Formatter/Xml.php

```diff
@@ -66,15 +66,15 @@ public function __construct($options = [])
                 'rootElement' => array_shift($args)
             ];
 
-            if (count($args)) {
+            if ($args) {
```

Comment:
I would rather change  it to `empty()` as more expressive

Reply:
updated.